src/components/Footer.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { GymName } from "../constants";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  fab,
+  faFacebookSquare,
+  faTwitterSquare,
+  faGooglePlusSquare,
+} from "@fortawesome/free-brands-svg-icons";
+import {
+  fas,
+  faLocationDot,
+  faHourglass,
+  faDumbbell,
+} from "@fortawesome/free-solid-svg-icons";
+const Footer = () => {
+  library.add(
+    fab,
+    faFacebookSquare,
+    faTwitterSquare,
+    faGooglePlusSquare,
+    fas,
+    faLocationDot,
+    faHourglass,
+    faDumbbell
+  );
```

Comment:
This will impact the performances because library for Font Awesome will be imported when rendering occurs.
Is there any reason why you have done this way?

Reply:
If the project gets bigger we can reuse icons into each component once they have been added to the library. But your concern of performance is noted and I will change the code.